e2e/util/logger.go

```diff
@@ -0,0 +1,41 @@
+package util
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"path/filepath"
+
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+)
+
+func CreateLogger(executableName string, verbose bool) *zap.SugaredLogger {
+	logFileName := fmt.Sprintf("%s.log", executableName)
+	logFilePath := filepath.Join(logFileName)
+	consoleConfig := zap.NewProductionEncoderConfig()
+	consoleConfig.EncodeTime = zapcore.ISO8601TimeEncoder
+	consoleConfig.EncodeLevel = zapcore.CapitalLevelEncoder
+	consoleConfig.CallerKey = zapcore.OmitKey
+	logfileConfig := zap.NewProductionEncoderConfig()
+	logfileConfig.EncodeTime = zapcore.ISO8601TimeEncoder
+	logfileConfig.EncodeLevel = zapcore.CapitalLevelEncoder
+	consoleEncoder := zapcore.NewConsoleEncoder(consoleConfig)
+	logfileEncoder := zapcore.NewConsoleEncoder(logfileConfig)
+	logfile, err := os.Create(logFilePath)
+	if err != nil {
+		log.Fatalf("Failed to create log file: %v", err)
```

Comment:
If we cannot create the log file, return the error. Helpers like this are more useful when they return an error. This let the caller handle the error int the best way. In our case it will panic.